doc/source/changelog.rst

```diff
@@ -2,6 +2,60 @@
 Changelog
 #########
 
+Schema Release ``v0.6.0a10``
+==========================
+
+Changes to ``ga4gh/schemas`` ``master`` branch since version ``0.6.0a9`` (Jan 23, 2016)
+
+Features:
+ * Remove feature_id from ExpressionLevel and addeed ability to seearch by Name field. `#818 <https://github.com/ga4gh/ga4gh-schemas/issues/818>`__  Impacts
+  * `POST /expressionlevels/search`
+  * `GET /expressionlevels/{id}`
+ - Added support for BigWig files in a new Continuous Data object `#802  <https://github.com/ga4gh/ga4gh-schemas/issues/802>`__ Adds the following new endpoints:
+  - Continuous Data: `POST /continuoussets/search`
```

Comment:
Single backticks don't work the same in RST as in Markdown. If you intend these to be inline monospace you'll need to add another backtick.

http://docutils.sourceforge.net/docs/user/rst/quickref.html

>`interpreted text` 	(see note at right) 	The rendering and meaning of interpreted text is domain- or application-dependent. It can be used for things like index entries or explicit descriptive markup (like program identifiers).